--- dico-2.11/configure.ac.orig	2021-06-19 07:10:22.576405899 +0200
+++ dico-2.11/configure.ac	2021-06-19 07:11:04.548583966 +0200
@@ -46,9 +46,9 @@
 LTDL_INIT([nonrecursive])
 
 dnl Checks for libraries.
-AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(nsl, gethostbyaddr)
-AC_CHECK_LIB(rt, nanosleep)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(gethostbyaddr, nsl)
+AC_SEARCH_LIBS(nanosleep, rt)
 
 dnl Checks for header files.
 AC_HEADER_DIRENT dnl not needed ?
